scripts/makeupdates

```diff
@@ -289,19 +289,10 @@ def copy_updated_files(tag, updates, cwd, builddir):
                 install_to_dir(gitfile, "usr/share/anaconda/dbus/services")
             elif gitfile.endswith(".conf"):
                 install_to_dir(gitfile, "usr/share/anaconda/dbus/confs")
-        elif gitfile.find('/') != -1:
-            fields = gitfile.split('/')
-            subdir = fields[0]
-            if subdir in ['po', 'scripts','command-stubs', 'tests',
-                          'docs', 'fonts', 'utils',
-                          'liveinst', 'dracut', 'data']:
-                continue
-            else:
-                sys.stdout.write("Including %s\n" % (gitfile,))
-                install_to_dir(gitfile, tmpupdates)
         else:
```

Comment:
What about removing this whole `else` part?

Reply:
Fixed.